korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-15552
-commithash:526c2d8d521343e5a29c2a1323925528cb94e873
+version:2.1.0-preview1-15559
+commithash:2c59937fbbe6f2ae5ceb6c5ff5a3bd35577d96ca
```

Comment:
@natemcmaster When does this happen?

Reply:
Nobody needs this, so it’s probaly just happening

Reply:
yeah, you can probably revert. This change will happen if you delete the file and re-run build.cmd, or if you run `build.cmd -Update`

Reply:
Yeah I ran build.cmd -u and it pulled in this version. Eventually this will need to happen on every PR that has a corresponding change to ANCM.

Reply:
build.cmd -u just updates build tools, not ANCM. To update ANCM, change this: https://github.com/aspnet/IISIntegration/blob/9c065bf1075af2092be468d75dcbb06c6f6dd49b/build/dependencies.props#L7